coil-base/src/main/java/coil/bitmap/BitmapPoolStrategy.kt

```diff
@@ -95,7 +95,7 @@ internal class SizeStrategy : BitmapPoolStrategy {
     override fun toString() = "SizeStrategy: entries=$entries, sizes=$sizes"
 
     companion object {
-        private const val MAX_SIZE_MULTIPLE = 8
+        private const val MAX_SIZE_MULTIPLE = 4
```

Comment:
This means only bitmaps at most 2x the size of the target size will be reused (we wouldn't want to reuse a 4mb bitmap for a 20kb image).